src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java

```diff
@@ -82,7 +84,7 @@ public void shouldReturnFailedFutureIfRequestHasNoTagId() {
     @Test
     public void shouldReturnFailedFutureIfStoredBidRequestHasNoImp() {
         // given
-        given(httpRequest.getParam(anyString())).willReturn("tagId");
+        given(httpRequest.getParam("tag_id")).willReturn("tagId");
```

Comment:
This line should be moved to `setUp()` method since it is required param for AMP. Fix all tests pls.